src/tests/ftest/pool/pool_svc.py

```diff
@@ -130,6 +129,8 @@ def test_pool_svc(self):
                         "DaosServerManager.stop_ranks([{}])".format(
                             pool_leader))
 
+                self.pool.wait_for_rebuild(to_start=False, interval=1)
```

Comment:
To verify that rebuild has completed, this should be called twice - once to verify rebuild has indeed started (`to_start=True`) and a second time to ensure it has completed (`to_start=False`).  Without verifying that rebuild has started you may get a false positive when detecting if rebuild has completed.
```suggestion
                self.pool.wait_for_rebuild(to_start=True, interval=1)
                self.pool.wait_for_rebuild(to_start=False, interval=1)
```

You may also want to consider adding a `rebuild_timeout: <seconds>` yaml entry under the `pool:` section in this test's yaml file.  See https://github.com/daos-stack/daos/blob/1bc2be36b3b14cdc12bf349e4f52ac9174bf540e/src/tests/ftest/util/test_utils_pool.py#L484 for more details.